models/balancer/arbitrum/balancer_v2_arbitrum_bpt_supply.sql

```diff
@@ -0,0 +1,514 @@
+{{ config(
+    schema = 'balancer_v2_arbitrum',
+    materialized = 'table',
+    file_format = 'delta',
+    alias = 'bpt_supply',
+    post_hook = '{{ expose_spells(\'["arbitrum"]\',
```

Comment:
do you eventually plan to union these all together into one spell?
if so, i'd recommend removing the expose spell config on each blockchain-specific one, then have only the cross-chain displayed on data explorer. otherwise, it's fine as-is

Reply:
Change to this?
```
post_hook = '{{ expose_spells("project",
                                                "balancer_v2",
                                                \'["thetroyharris"]\') }}'
```

Reply:
i would remove the post hook config altogether, if you plan to later union cross-chain and have that be the main spell